src/kirby/components/icon/kirby-icon-settings.ts

```diff
@@ -46,5 +46,20 @@ export const kirbyIconSettings: IconSettings = {
     { name: 'verify', svg: 'assets/kirby/icons/svg/verify.svg', unicode: 0xe927 },
     { name: 'warning', svg: 'assets/kirby/icons/svg/warning.svg', unicode: 0xe928 },
     { name: 'write-message', svg: 'assets/kirby/icons/svg/write-message.svg', unicode: 0xe929 },
+    { name: 'accounts-outline', svg: 'assets/kirby/icons/svg/accounts-outline.svg', unicode: 0xe900 },
```

Comment:
I spot some duplicates entries here. `accounts-outline` is defined on line 7. 

Please remove duplicates.

Reply:
Nice spotted :)